packages/components/src/popover/style.scss

```diff
@@ -2,6 +2,7 @@ $arrow-triangle-base-size: 14px;
 
 .components-popover {
 	z-index: z-index(".components-popover");
+	will-change: transform;
```

Comment:
@tyxla since the number of popover is limited I don't think this is a big issue, BUT it might actually not be needed since popovers seem to get a translateZ for free 😄

Reply:
@tyxla I have reverted the change since the translateZ does the job, however it seems to trigger repaint of some of the elements inside the block toolbar which is ... weird 😅

Reply:
As @corentin-gautier mentioned, technically we shouldn't need it because of the `translateZ()` value that framer motion already adds — in that sense, I didn't expect the browser to trigger repaints as reported in the previous message 🤷 although browser heuristics about when to create a new composite layer can change and evolve in time, so we're never really guaranteed.

Regarding the when we should add/remove this prop, it's worth mentioning that [the `Popover` component should be rendered only when the popover is actually visible](https://github.com/WordPress/gutenberg/blob/f0fe8f64fe0afef0f383ffa8ea6320d1c774f600/packages/components/src/popover/README.md?plain=1#L11) — therefore, it seems OK to me that we keep that CSS rule as always ON: it's only rendered when the popover is open.

Reply:
@ciampo do you recommand i add it back then ?

Reply:
It would be good if you could investigate a bit more into why removing `will-change: transform` causes paint flashes in the toolbar, and see if we can achieve the desired result in any other way.

But if after the investigation the only guaranteed way to avoid paint flashes if by having `will-change: transform` always applied, then yes — I would recommend you add it back at that point

Reply:
Have we tried setting `contain: paint` on the toolbar buttons, out of curiosity?

> There are other tricks to avoid paint like using backface-visibility: hidden; so that's an option too :)

Yup, although that doesn't really change the fact that we'd be "forcing" the popover to be on its own composite layer — so `will-change: transform` seems like a better (more semantic) choice

Reply:
🤷  Let's stick to `wil-change: transform` then! Thank you for the investigation :) 

If you ever feel like digging more into this and come up with more insights, I'm always eager to see how we can improve this component further!